arch/arm/boot/dts/overlays/README

```diff
@@ -695,6 +695,14 @@ Load:   dtoverlay=dpi18
 Params: <None>
 
 
+Name:   dpi18cpadhi
+Info:   Overlay for a generic 18-bit DPI display (in 'mode 6' connection scheme)
+        This uses GPIOs 0-9,12-17,20-25 (so no I2C, uart etc.), and activates the output
```

Comment:
This line is too long - there is a hard 80-character limit.

Reply:
Undesrtood. Will fix this.